### ISSUE TITLE
Can't login, error 5002, can't validate IP address

### ISSUE THREAD

**Issue by ragazzojp** [openmediavault/openmediavault#884]
### Description of issue/question

Frequently, let's say 30%-50% of the times, I try to login to my OMV server via its domain name (nas.local) but after providing correct username and password I receive this error:

```
Error #5002:
OMV\ErrorMsgException: Invalid IP address in /usr/share/php/openmediavault/session.inc:188
Stack trace:
#0 /usr/share/php/openmediavault/session.inc(212): OMV\Session->validateIpAddress()
#1 /var/www/openmediavault/index.php(34): OMV\Session->validate()
#2 {main}
```

This seems that this security/validation mechanism gets confused by the fact that the browser may use different IP addressed to communicate to the OMV server. This may be due to the presence of IPv6.

### Steps to reproduce issue

I have public IPv6 on my network, so both the computer I'm testing from and the NAS have 3 valid addresses:

1) IPv4 (196.168.0.x)
2) local IPv6 (fe80::x)
3) public IPv6 (2a02::x)

How to replicate: just login. If it works, retry. The problem is sporadic but frequent.

Browser used: Firefox 80, on both Linux and Windows 10.

To workaround the problem, I have to use an IP address explicitly in the browser. This is tested using IPv4 only.

### Versions report

4.x

**Comment by votdev** [comment]
Right, i'm also annoyed by this behaviour from time to time. I think locking a session to an IP is not a good idea anymore nowadays. I will remove this.

**Comment by votdev** [comment]
Fixed in `openmediavault 5.5.19`.
